gatsby_fp/src/components/fieldComponents/genInfoTable.js

```diff
@@ -0,0 +1,122 @@
+import React from "react"
+
+export default function GenInfoTable({
+  school,
+  title,
+  imgsrc,
+  building,
+  room,
+  phone,
+  fax,
+  username,
+  website,
+}) {
+  return (
+    <>
+      <div className="table_wrapper">
+        <div className="table_wrapper_inner">
+          <table
+            cellPadding="5"
+            cellSpacing="0"
+            border="0"
+            className="bioregion"
+            width="100%"
+          >
+            <tbody>
+              <tr>
+                <td valign="top" align="left" width="160px">
+                  <img
+                    src={`data:image/jpeg;base64,${imgsrc}`}
+                    className="bioimage"
+                    style={{ maxWidth: "150px" }}
```

Comment:
This specific instance of inline styling might bite us sooner, since the `style` attribute takes precedence over anything else. Let's move the `maxWidth` into the CSS (using `.bioimage` to target it) instead of using the inline `style` attribute for it.

Reply:
Good point. We'll want it near line 27 of facultyprofile.css (site.css should not be touched, so that we can more easily merge in changes from upstream).